odoo/addons/test_main_flows/static/tests/tours/main_flow.js

```diff
@@ -195,12 +195,10 @@ tour.register('main_flow_tour', {
     position: 'right',
     run: "text 1",
 }, {
-    trigger:  ".modal-footer .btn-primary:first",
-    content: _t('Save & Close'),
-    position: 'bottom',
-}, {
+    trigger: 'label:contains("Vendor Taxes")',
+    // click somewhere else to exit cell focus
+},{
```

Comment:
Don't make changes in test files in independent commit. If you need to make adaptation in a test, do it in the same commit who introduces these changes.
The purpose to this is to have each commit able to pass the tests without need to wait further commit.